app/models/champs/linked_drop_down_list_champ.rb

```diff
@@ -89,6 +89,17 @@ def has_secondary_options_for_primary?
     primary_value.present? && secondary_options[primary_value]&.any?(&:present?)
   end
 
+  def contains_options?(options)
+    options.include?(primary_value) || options.include?(secondary_value)
+  end
+
+  def remove_options(options)
+    update_column(:value, [
+      options.include?(primary_value) ? "" : primary_value,
+      options.include?(secondary_value) ? "" : secondary_value
+    ].to_json)
```

Comment:
la méthode est plus maligne que celle sur la liste simple. Du coup, je serai pour shooter les 2 choix par cohérence

```suggestion

  def remove_options
    update_column(:value, [ "" , "" ].to_json)
```

Reply:
ok